packages/shared/src/wallet/FormattedBalance.tsx

```diff
@@ -13,13 +13,22 @@ export interface FormattedBalanceProps extends withClasses<'balance' | 'symbol'>
     decimals?: number
     significant?: number
     symbol?: string
+    minimumBalance?: number
     formatter?: (value: BigNumber.Value, decimals?: number, significant?: number) => string
 }
 
 export const FormattedBalance: FC<FormattedBalanceProps> = (props) => {
-    const { value, decimals, significant, symbol, formatter = (value) => value } = props
-    const formatted = formatter(value ?? '0', decimals, significant)
+    const { value, decimals, significant, symbol, minimumBalance, formatter = (value) => value } = props
+    const formatted =
+        Number(formatter(value ?? '0', decimals, significant)) === 0
```

Comment:
plz save `formatter(value ?? '0', decimals, significant)` result as variable

it's re-calculating

Reply:
plz use https://github.com/DimensionDev/Maskbook/blob/develop/packages/web3-shared/base/utils/number.ts

e.g: `isZero(balance)` (n === 0) /  `isLessThan(balance, minimumBalance)`